ghu_web/ghu_main/migrations/0015_auto_20171018_2210.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.1 on 2017-10-19 02:10
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('ghu_main', '0014_orgprofile_summary'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='orgprofile',
+            name='summary',
+            field=models.CharField(max_length=256, null=True),
+        ),
+    ]
```

Comment:
this is lot of migrations. could we squash them into one? you should be able to do this with

    $ python3 manage.py squashmigrations ghu_main 0007 0015

then you can delete the old migrations and then rename the migration in `ghu_main/migrations` to something like `0007_orgprofiles.py`